x-pack/plugins/maps/server/plugin.ts

```diff
@@ -197,6 +198,7 @@ export class MapsPlugin implements Plugin {
             read: [MAP_SAVED_OBJECT_TYPE, 'index-pattern', 'query', 'tag'],
           },
           ui: ['show'],
+          api: ['savedQuery:read'],
```

Comment:
Does this mean you can only access maps application if you have the role 'savedQuery:read'? I am not sure that restriction makes sense.

Reply:
No, it means that by having access to the maps application you are automatically granted access to APIs that require `savedQuery:read` privileges.

Reply:
thanks for clarifying